scripts/trigger-mobile-metrics.py

```diff
@@ -36,7 +36,7 @@ def TriggerWorkflow(token, commit, publish):
       print("Started run_android_navigation_benchmark: %s" % response_dict)
 
 def TriggerJob(token, commit, job):
-    url = "https://circleci.com/api/v1.1/project/github/mapbox/mobile-metrics/tree/main"
```

Comment:
nice catch :) looks like it was [find and replace all](https://github.com/mapbox/mapbox-navigation-android/commit/b8233660c7f09754f58c85874b3709fda499b26f)